ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -355,6 +355,24 @@ type ${project.artifactId}_unitfile_config_socket = {
     true;
 };
 
+@documentation{
+the [Path] section
+https://www.freedesktop.org/software/systemd/man/systemd.path.html
+}
+type ${project.artifactId}_unitfile_config_path = {
+    'PathExists' ? absolute_file_path
+    'PathExistsGlob' ? string
+    'PathChanged' ? absolute_file_path
+    'PathModified' ? absolute_file_path
+    'DirectoryNotEmpty' ? absolute_file_path
+
+    'Unit' ? string
+    'MakeDirectory' ? boolean
+    'DirectoryMode' ? type_octal_mode
+    'TriggerLimitIntervalSec' ? string
```

Comment:
```suggestion
    'TriggerLimitIntervalSec' ? long(0..)
```

Reply:
The documentation suggests users put a unit prefix (e.g. 1s or 1us): https://www.freedesktop.org/software/systemd/man/systemd.path.html
Do we have a type which supports this?

Reply:
For simplicity we normally only support the base unit (in this case seconds).